hr_employee_calendar_planning/views/resource_calendar_views.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Creu Blanca
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="resource_calendar_form" model="ir.ui.view">
+        <field name="model">resource.calendar</field>
+        <field name="inherit_id" ref="resource.resource_calendar_form" />
+        <field name="arch" type="xml">
+            <notebook position="inside">
+                <page name="employee_calendar" string="Employee Calendars">
```

Comment:
Adding a new tab doesn't quite convince me, maybe you could move `employee_calendar_ids` field to another place (inside the tab it was previously in) to be the full size of the tab. What do you think @pedrobaeza ?

Reply:
Yes, a whole new tab just for this seems over-killing.

Reply:
What do you think about leaving the label and setting a colspan? Looks like this:
![employee_calendar_planning_form](https://github.com/OCA/hr/assets/93716681/5adc1473-ee8d-4dbc-817e-5de765acfca1)

Reply:
It seems it overflows a bit the limits.

Reply:
Yes, I know what you mean. But to me, this seems like the way Odoo is treating fields like this in views, see eg [here](https://github.com/odoo/odoo/blob/d4b366f2d741f2087a947d61d7e0618492cc2bf1/addons/account/views/partner_view.xml#L207). Defining custom css just for this view seems a bit unnecessary. Alternatively, we could place it in a sheet, like this:
![image](https://github.com/OCA/hr/assets/93716681/89d27818-0f3d-439d-b48a-b55c01074403)

Reply:
OK, that can be another option.